packages/flutter/test/material/progress_indicator_test.dart

```diff
@@ -548,7 +535,7 @@ void main() {
         find.byType(LinearProgressIndicator),
         paints
         ..rrect(
-          rrect: RRect.fromLTRBR(0.0, 0.0, 100.0, 4.0, const Radius.circular(10.0)),
+          rrect: RRect.fromLTRBR(29.0, 0.0, 100.0, 4.0, const Radius.circular(10.0)),
```

Comment:
The current LinearProgressIndicator renders the track rect as background track and draws active track on top of it. 

In the new M3 design, active track and back inactive track are split into two and there is a gao. As result first rect starts from 29.0 left instead of 0.0 left.

Reply:
Ah that makes sense. Thanks for the explanation!